util/syn_yosys.sh

```diff
@@ -10,59 +10,139 @@
 # The following tools are required:
 #  - sv2v: SystemVerilog-to-Verilog converter from github.com/zachjs/sv2v
 #  - yosys: synthesis tool from github.com/YosysHQ/yosys
+#  - Cadence Conformal
 #
 # Usage:
-#   syn_yosys.sh 2>&1 | tee syn.std
+#   ./syn_yosys.sh |& tee syn.std
```

Comment:
Ok about the added `./` (I assume you're trying to make it clear that you have to run the script from its directory). Not so keen on the change to how redirection works: the previous version worked in any posix shell. The new version just works in bash.

Reply:
Good point, thanks! I'll create a new PR soon based on the latest syn_yosys.sh with all your suggestions included.